src/mono/netcore/System.Private.CoreLib/src/System/TypedReference.cs

```diff
@@ -87,20 +86,17 @@ public unsafe static object ToObject (TypedReference value)
 			}
 		}
 
-		[CLSCompliant (false)]
 		public static Type GetTargetType (TypedReference value)
 		{
 			return __reftype (value);
 		}
 
-		[CLSCompliant (false)]
 		public static RuntimeTypeHandle TargetTypeToken (TypedReference value)
 		{
 			return __reftype (value).TypeHandle;
 		}
 
-		[CLSCompliant (false)]
-		public unsafe static void SetTypedReference (TypedReference target, Object value)
+		public unsafe static void SetTypedReference (TypedReference target, Object? value)
```

Comment:
nit: object?

Reply:
Happy to use `object` over `Object`. Didn't make the change here because wasn't sure if `Object` was a convention / style I'd be changing.